src/uwtools/tests/api/test_chgres_cube.py

```diff
@@ -1,26 +1,29 @@
-# pylint: disable=missing-function-docstring,protected-access
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
 
 import datetime as dt
+from pathlib import Path
```

Comment:
Like the library changes, the test changes are mostly repetitions of a common theme.